public/ge0.html

```diff
@@ -20,97 +20,207 @@
   <title>${title}</title>
 </head>
 
+<script>
+function openStore() {
+  window.location='/get.html';
+}
+
+function triggerApp() {
+  window.location='om:/${path}';
+  setTimeout(openStore, 700);
+}
+
+function copyCoords() {
+  el = document.getElementById("notif");
+  el.className = "show";
+  setTimeout(function(){ el.className = el.className.replace("show", ""); }, 1500);
+  navigator.clipboard.writeText("${lat}, ${lon}");
+}
+</script>
+
 <body>
   <div id="header">
-    <span id="name">${name}</span>
-    <span>
-      <a id="open" href="om:/${path}" class="link">
-        Open in Organic Maps
-      </a>
-      <a href="https://omaps.app/get" target="_blank" class="link">
-        Install Organic Maps
+    <div id="container">
+      <a href="https://organicmaps.app/">
+        <img id="logo" src="/icons/logo-icon.svg"/>
       </a>
-      <a href="https://www.openstreetmap.org/?mlat=${lat}&mlon=${lon}#map=${zoom}/${lat}/${lon}" target="_blank" class="link">
-        See on OpenStreetMap
-      </a>
-    </span>
+      <span id="name">${name}</span>
+      <div>
+        <a id="open" onclick="triggerApp()" href="om:/${path}" class="linkcolour link">
+          Open in Organic Maps
+        </a>
+        <a id="install" href="https://omaps.app/get" target="_blank" class="linkcolour link">
+          Install Organic Maps
+        </a>
+        <a href="https://www.openstreetmap.org/?mlat=${lat}&mlon=${lon}#map=${zoom}/${lat}/${lon}" target="_blank" class="linkcolour link">
+          See on OpenStreetMap
+        </a>
+        <div class="link">
+          <a href="geo:${lat},${lon}?Z=${zoom}" target="_blank" class="linkcolour">
+            ${lat}, ${lon}
+          </a>
+          <div class="green">|</div>
+          <a href="#" onclick="copyCoords()" class="linkcolour">
+            Copy
+          </a>
+        </div>
+      </div>
+    </div>
   </div>
+  <div id="notif" class="notif">Coordinates copied to clipboard.</div>
   <div id="map" class="map"></div>
   <script type="text/javascript">
-    const isiOS = navigator.platform.substr(0, 2) === 'iP' ||  // iPhone, iPad, iPod, including simulators.
+    var isiOS = navigator.platform.substr(0, 2) === 'iP' ||  // iPhone, iPad, iPod, including simulators.
       (navigator.userAgent.includes('Mac') && 'ontouchend' in document);  // iPad on iOS 13.
-    const isAndroid = !isiOS && /(android)/i.test(navigator.userAgent);
-    const isDesktop = !isiOS && !isAndroid;
+    var isAndroid = !isiOS && /(android)/i.test(navigator.userAgent);
+    var isDesktop = !isiOS && !isAndroid;
+    var isMobile = isiOS || isAndroid;
 
     if (isDesktop)
       document.getElementById('open').remove();
-
-    const lat = ${ lat };
-    const lon = ${ lon };
-    const zoom = ${ zoom };
-    const map = L.map('map').setView([lat, lon], zoom);
-    L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+      
+    if (isMobile)
+      document.getElementById('install').remove();
```

Comment:
Why is it removed on mobiles? How it will appear if the app is not installed?

Reply:
On mobile, only the 'Open in OrganicMaps' will appear:

-  if the app is installed, it will open the location in OM
- if it isn't, it will direct the user to install OM by opening get.html

it looks like this, no matter if the app is installed:
![Screenshot_20220310-222510_Mull](https://user-images.githubusercontent.com/26939824/157765231-c95953f2-f774-42d5-bf10-cbc5719d3899.png)

it'd be nice to change it to 'Install OrganicMaps' if it isn't detected as installed, but afaik this isn't possible :(